### ISSUE TITLE
Blender Crash on "Merge Armature" button press under "Custom Model Creation"

### ISSUE THREAD

**Issue by Isuress** [absolute-quantum/cats-blender-plugin#621]
When merging any armature with my avatar's primary armature, Blender crashes.
I'm using the latest released Blender on Steam (_v3.6.0_).
The only way I'm able to merge armatures is if I launch the portable version of **Blender v3.5.1** and merge then export as needed.

I shouldn't have to do this every time I want to merge an armature or export an avatar.
CATs has worked with bugs up until this version; although we haven't had an "official" update since 2021.

Context menu and settings:
![image](https://github.com/absolute-quantum/cats-blender-plugin/assets/7573781/55a5629d-9c7e-4dfd-b13e-ee70a33ff8eb)

Crash log:
[Yugi & Miyo - Blend - 00-052 - Import SimpleTightDress, & MaterialFix (TinaQT Fitted).crash.txt](https://github.com/absolute-quantum/cats-blender-plugin/files/11963113/Yugi.Miyo.-.Blend.-.00-052.-.Import.SimpleTightDress.MaterialFix.TinaQT.Fitted.crash.txt)

Additionally, when pressing "install development version" I get this error.
![image](https://github.com/absolute-quantum/cats-blender-plugin/assets/7573781/e3f4a7e9-481b-4383-9612-64162d4752ea)

**Comment by woolstar2000** [comment]
Yep, I had the same issue. Seems to be an issue with 3.6, I just had to do all my work by exporting to FBX, going to blender 3.5, merging there, then reimporting to 3.6

**Comment by MonsterBunny240** [comment]
I ligit thought i was the only one having this problem does anyone know how to fix it :(.

**Comment by eunomia-jpg** [comment]
> I ligit thought i was the only one having this problem does anyone know how to fix it :(.

Just downgrade to Blender 3.5. It's only a 3.6 issue.

**Comment by Isuress** [comment]
> > I ligit thought i was the only one having this problem does anyone know how to fix it :(.
> 
> Just downgrade to Blender 3.5. It's only a 3.6 issue.

Downgrading is a workaround, not a fix. You should not have to hinder the application for a fairly old and well-used plugin.
It seems the developer edition of the plugin is getting updates; there should be a push to release soon as it's become unusable.

**Comment by wierli** [comment]
same issue, pls fix it on 3.6 version.
